documentation/source/users/rmg/releaseNotes.rst

```diff
@@ -77,12 +77,13 @@ Date: July 7, 2022
 - QM Corrections
     - Added frequency scaling factors AEC and BAC for ωB97X-D3/def2-TZVP and B97-D3/def2-mSVP
     - Added notebooks demonstrating how to update AEC and BAC
-    - Updated CCSD(T)-F12 AECs and BACs
+    - Updated AECs for CCSD(T)-F12/cc-pVXZ-F12 st X = D, T
     - Added frequency factor for b2plypd3/def2tzvp
     - Updated BAC using correct frequency scaling factor for:
-        - CCSD(T)-F12/cc-pVTZ-F12
-        - wb97xd3/def2-tzvp
-        - b97-d3/def2-msvp
+        - wB97X-D3/def2-TZVP
+        - B97-D3/def2-mSVP
+        - CCSD(T)-F12/cc-pVTZ-F12//ωB97X-D3/def2-TZVP
+        - F12/cc-pVDZ-F12//ωB97X-D3/def2-TZVP
```

Comment:
Thanks for addressing my minor comments :) Could you double check this line? It seems that the `CCSD(T)-` was accidentally dropped. It should say the same method as the line above: `CCSD(T)-F12/cc-pVDZ-F12//ωB97X-D3/def2-TZVP`. Thanks!

Reply:
Oof, that was clumsy of me. Should be fixed now!